cli/internal/config/config.go

```diff
@@ -53,9 +61,13 @@ func InitConfig(cfgFile string) {
 	viper.SetDefault("debug", false)
 	viper.SetDefault("verbose", false)
 	viper.SetDefault("apiURL", client.BASE_URL_V2)
+	viper.SetDefault("telemetry.writeKey", "")
```

Comment:
Why not set it to the `TelemetryWriteKey` var?
That way you can also avoid the check in the function that falls back to this var anyway,

Reply:
Thanks for this, oversight on my part.